processor/processor.go

```diff
@@ -191,7 +199,55 @@ func ProcessConstants() {
 			printTrace(fmt.Sprintf("milliseconds build language features: %d", makeTimestampMilli()-startTime))
 		}
 	} else {
-		printTrace("configured to lazy load language features")
+		if Trace {
+			printTrace("configured to lazy load language features")
+		}
+	}
+}
+
+// Configure and setup any count-as params the use has supplied
+func setupCountAs() {
+	for _, s := range strings.Split(CountAs, ",") {
+		t := strings.Split(s, ":")
+		if len(t) == 2 {
+
+			identified := false
+
+			// There are two cases here.
+			// first is they provide the name "Cargo Lock"
+			// second is that the user supplies the extension EG wsdl
+			// we should support BOTH cases
+			// always remember we only need to validate t[1] as thats the one
+			// that tells us where we are trying to map
+
+			// See if we can identify based on language name which is the most
+			// reliable as the name should be unique
+			for name := range languageDatabase {
+				if strings.ToLower(name) == strings.ToLower(t[1]) {
+					ExtensionToLanguage[t[0]] = []string{name}
+					identified = true
+					if Debug {
+						printDebug(fmt.Sprintf("set to count extension: %s as language %s by language", t[0], name))
+					}
+				}
+			}
+
+			// If the above did not work, its a matter of extension match
+			// note that this is less reliable as some languages share extensions
+			if !identified {
+				for name, value := range languageDatabase {
+					for _, ext := range value.Extensions {
+						if strings.ToLower(ext) == strings.ToLower(t[1]) {
```

Comment:
Since we're doing this after building `ExtensionToLanguage`, we should be able to look up from that rather than iterating through the entire language list for every remapping.

Reply:
Good point. I moved it out of the previous loop because it would have made it do it every time. Didn't even think to use the lookup itself.